packages/electron/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "dev": "rm -rf dist && tsc --watch",
     "build": "rm -rf dist && tsc",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "node ../../scripts/check-command.js && npm run build"
```

Comment:
If you really want to force the use of pnpm then there is a better option than creating a custom script
`package.json` has an option to set package manager explicitly, it would look like that:
```json
"packageManager": "pnpm@7.0.0",
```
And if the user's package manager differs, the script will refuse to execute:

![image](https://user-images.githubusercontent.com/40315177/182020658-5233d600-ab25-4acb-b189-8760dc7fedf6.png)

Though [this option is experimental](https://nodejs.org/api/corepack.html#configuring-a-package), it still works fine for me 😄 

With that option you can remove the script and command chaining

Reply:
Wow what a great idea!

The purpose of `scripts/check-command.js` is to prevent me from using `npm publish` by mistake, this is because the dependencies version of `"vite-plugin-electron-renderer"` is `"workspace:*"`, so I must use `pnpm publish`.

Early on, I made this mistake once. 😅 #43

I tried adding `packageManager` to `packages/electron/package.json`, which doesn't seem to prevent me from using `npm publish`. But it is necessary to put it in `package.json` in the root directory. 😄

Reply:
Oh, got it 😆 

In that case, I don't know if adding `packageManager` will help. As I remember it still allows `npm` but blocks any other package manager. Though it may provide some help.

P.S. I just simply don't have `npm` installed 👀